addons/website/static/src/scss/website.scss

```diff
@@ -927,11 +930,8 @@ header {
     #top_menu_collapse {
 
         &.collapsing, &.show {
-            // Note: position relatively to the header instead of the viewport
-            // because fixed position cannot work inside an element whose CSS
-            // transform is different to none, which the header element is
-            // because of header effects.
```

Comment:
I see you adapted to this below but I am bit scared to change this in stable. Do we really have to switch to fixed?

Reply:
If we want to be sure that the "off-canvas" menu is always in the top corner (left or right) of the page, there is no other solution. The absolute position was not robust enough, we saw it as soon as the designers added new headers.

Reply:
Ok :+1: